tests/test_config.py

```diff
@@ -1,7 +1,17 @@
-import distutils.util
 import os
 
-DEV_MODE = bool(distutils.util.strtobool(os.environ.get("DEV_MODE", "False")))
+
+def strtobool(val: str):
+    normalized_val = val.lower()
+    if normalized_val in ('y', 'yes', 't', 'true', 'on', '1'):
+        return True
+    if normalized_val in ('n', 'no', 'f', 'false', 'off', '0'):
+        return False
+    else:
+        raise ValueError(f"invalid truth value {val}")
+
+
+DEV_MODE = strtobool(os.environ.get("DEV_MODE", "False"))
```

Comment:
DEV_MODE isn't used, lets remove it, instead of introducing new code

Reply:
Will do